ports.mk

```diff
@@ -70,6 +70,9 @@ ifdef DYNAMIC_MODULES
 	rm -rf "$(DESTDIR)$(libdir)/scummvm/"
 endif
 
+bundle-mingw: install-strip
+	@cp -vu $(shell ldd "$(DESTDIR)$(bindir)"/$(EXECUTABLE) | grep -i mingw | cut -d">" -f2 | cut -d" " -f2 | sort -u) "$(DESTDIR)$(bindir)"
```

Comment:
I am not sure to see how ldd works here as it's a Windows executable. Does it work when cross compiling?

Reply:
I didn't test. I'll try to test it tomorrow.

Reply:
No, it doesn't work on linux. It is meant for MSYS2 environment.

Finding the dependencies on linux is complicated. Can be done [using objdump](https://stackoverflow.com/a/58671570/764870), but this only gives a raw list. You need to locate each dll from the list on the path.

But anyway, the cross compilation is usually statically linked, isn't it? So this target is not needed at all for this kind of build.

Reply:
I'm not sure if this should be part of scummvm at all

Reply:
Why not? It's just an extra target, doesn't affect anyone who doesn't use it directly.

See the [MinGW instructions](https://wiki.scummvm.org/index.php/Compiling_ScummVM/MinGW-w64). They're rather scary.

I'm trying to simplify them by [providing the needed curl package](https://github.com/msys2/MINGW-packages/pull/8469), and creating a simple target for bundling the required dlls (`make bundle-mingw`).

After both are done, the instructions can be reduced to more or less 5 lines.

Reply:
there is nothing scary there. it's just moving step from wiki to makefile.
it's adding mess. at least should be pointed as only MinGW windows only to prevent confusion

Reply:
I agree with @orgads here. We have other targets with platform-specific commands (e.g. ScummVMDockTilePlugin further down uses the macOS ``lipo`` command, iphonebundle uses the ``ldid`` homebrew tool for iOS, osxsnap uses the macOS ``hdiutil`` tool. I don't see why this is different, it's a Windows-only target